Libplanet.Tests/Net/PeerTest.cs

```diff
@@ -12,6 +12,8 @@ public class PeerTest
     {
         public static IEnumerable<object[]> GetPeers()
         {
+            var signer = new PrivateKey();
+            var ver = new AppProtocolVersion(signer, 1);
```

Comment:
I guess it would be better if we provide a factory method like `Block<T>.Mine()` instead of constructer.

Reply:
You mean it would be better if we can omit the first argument, `signer` here?

Reply:
Ah, I understood.  You mean *this* overload of the constructor misleads code readers into considering the given `PrivateKey` will be actually stored as a part of `AppProtocolVersion`'s internal state.  Fair enough.  I will turn this into a static method instead of a constructor.